hugo/content/about.md

```diff
@@ -0,0 +1,73 @@
+---
+title: About
+toc: true
+---
+
+# About the project
+
+Mumble is a free, open source, low latency, high quality voice chat application.
+
+## Source Code & Development
+
+You find the Source Code and further instructions regarding Building and Development in our [GitHub Repo](https://github.com/mumble-voip).
+
+## Licenses
+
+<!-- https://github.com/mumble-voip/mumble/blob/master/LICENSE -->
+
+<!-- more licenses? should be as correct and complete as possible -->
+
+# Who we are
+
+<!-- Title could also be: Developers, Team etc. -->
+
+<!-- improve -->
+
+We are a team of voluntary developers - both professional and non-professional coders - from different countries - including Germany, Italy, USA and more.
+
+<!--
+Link to Github Team page:
+https://github.com/orgs/mumble-voip/people
+
+-->
+
+<!-- Distinction between active and non-active developers? -->
+
+<!-- could contain more blabla about vision, free-software dedication etc. -->
```

Comment:
No real need to do that. We could instead link to a good website explaining what Open Source is about.

Reply:
Well, [wikipedia](https://en.wikipedia.org/wiki/Open-source_model) does not seem to be ideal...
if you know a good website.

Otherwise I will simply leave this out.

Reply:
I think https://en.wikipedia.org/wiki/Open-source-software_movement is not too bad for this purpose :thinking:

Reply:
What about calling Mumble `Free Software` and link to the [Wikipedia article](https://en.wikipedia.org/wiki/Free_software)?

Reply:
To be really correct, Mumble is FOSS (Free and Open Source Software). That basically covers everything. Thus https://en.wikipedia.org/wiki/Free_and_open-source_software is probably the most correct thing to link here :thinking:

Reply:
I just saw that you already describe mumble as "Mumble is a free, open source, low latency, high quality voice chat application.", so how should I include it?
For now I have only mentioned it in the section about "Licenses".

Reply:
You could simply turn "free, open source" into a link to that Wikipedia page :thinking: